DEVELOPING.md

```diff
@@ -198,3 +206,51 @@ hab pkg install -u http://localhost -z <auth-token> <package>
 Initially, you will get a `Package Not Found` error.  Wait for a bit (the package will get synchronized in the background) and try again - this time the install should succeed!
 
 *NOTE*: It is important to understand how the upstream cache is working. Packages that are requested (either via a `hab pkg install`, or even searching or browsing packages in the Web UI) in the local on-premise depot that have newer (or existing) versions in the upstream in the *stable* channel are marked for retrieval in the background. It is only after the background retrieval of the package succeeds that the package then becomes available in the local instance. If there is any failure to retrieve or submit the package, the next retrieval attempt will be triggered only by another request for that package. This functionality is new, and will be refined over time.
+
+#### Preparing to run local builds
+
+Initially, your depot will be empty, which means you won't be able to run a successful build. Minimally, you'll need to upload the current, stable version of `core/hab-backline` (which you'll have installed locally as a result of entering the studio), but to upload `hab-backline`, you'll need to create an origin named `core` as a home for it. Follow these steps to prepare an empty, upstream-enabled depot to run successful builds:
+
+  1. Ensure you've exported environment variables for `HAB_ORIGIN` and `HAB_AUTH_TOKEN` [as described above](#studio-requirements) that correspond with your account **on the public Builder service**.
+
+  1. Download the public and signing keys for the `core` origin from public Builder:
+
+    ```
+    hab origin key download core
+    hab origin key download core --secret
+    ```
+
+  1. Browse to your local instance of Builder UI (e.g., http://localhost), sign in, navigate to your Profile and generate a new personal access token (if you haven't already).
+
+  1. Since you no longer need to interact with public Builder, export a set of environment variables to point the `hab` client to your new, local instance of Builder:
+
+      ```
+      export HAB_ORIGIN=<YOUR_PERSONAL_ORIGIN> # If you've created one
+      export HAB_ORIGIN=<YOUR_PERSONAL_ACCESS_TOKEN>
```

Comment:
Couple of comments here: 1) the export presumably should be HAB_AUTH_TOKEN instead of HAB_ORIGIN  2) we should be clear not to do this within the studio, as it would then start pinging prod depot with the a bad auth token